std/exception.d

```diff
@@ -1533,21 +1533,6 @@ version (StdUnittest)
     assert( doesPointTo(cast(int*) s, i));
     assert(!doesPointTo(cast(int*) s, j));
 }
-@safe unittest //more alias this opCast
-{
-    void* p;
-    struct A
-    {
-        void* opCast(T)() if (is(T == void*))
-        {
-            return p;
-        }
-        alias foo = opCast!(void*);
-        alias foo this;
-    }
-    assert(!doesPointTo(A.init, p));
-    assert(!mayPointTo(A.init, p));
-}
 
```

Comment:
https://github.com/dlang/phobos/pull/8671 changed this unit test to make `A` a `struct` instead of a `class`.

Reply:
Yes, alias this for classes is now deprecated.